### ISSUE TITLE
Add completed filter

### ISSUE THREAD

**Issue by jrobcodes** [GuildCrafts/floworky#8]
As a user, I want to a toggle to be able to filter my items by completed, active, and all, so that I can create a focused view of my items.
- Add filter to navigation bar

**Comment by Moniarchy** [comment]
@EthanJStark and I can finish this if necessary

**Comment by jrob8577** [comment]
IIRC this is intended behavior, but that was a conversation we had last week so I can't remember.  @Moniarchy do you recall?

**Comment by Moniarchy** [comment]
Yes, this is a known issue and last week we(last week's floworky team)  were speculating that this behavior is something that could potentially be solved under the issue we're taking on this week, Ethan.
